examples/recording_service/pluggable_storage/c/FileStorageWriter.c

```diff
@@ -35,8 +36,24 @@
 
 #define NANOSECS_PER_SEC 1000000000ll
 #define FileStorageWriter_FILE_NAME_MAX 1024
+#define FileStorageWriter_INFO_EXT ".info"
 #define FILENAME_PROPERTY_NAME "example.c_pluggable_storage.filename"
 
+int RTI_fopen(FILE **file, const char *file_name, const char *mode)
```

Comment:
Maybe we should move this to a FileStorageUtils.c file that you share, so you don't need to copy the implementation.